.python-version

```diff
@@ -0,0 +1,3 @@
+3.6.9
+3.7.7
+3.8.2
```

Comment:
It looks like these are almost the most recent minor versions of `3.6`, `3.7`, and `3.8`, but I see releases for `3.6.10` and `3.8.3`.  What's the reasoning behind choosing these specific versions, and is the idea to increment them as new minor versions are released?

Reply:
I chose these based on what is currently available via `pyenv`. You're right about `3.6.10`, we can update to that, but `3.8.3` is not available via `pyenv` yet